released/packages/coq-hott/coq-hott.8.11/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Jason Gross <jgross@mit.edu>"
+homepage: "http://homotopytypetheory.org/"
+bug-reports: "https://github.com/HoTT/HoTT/issues"
+license: "BSD-2-Clause"
+build: [
+  ["bash" "-c" "./autogen.sh -skip-submodules || :"]
```

Comment:
Is the `-skip-submodules` argument really used when the code is delivered by tarball? I thought that had to do with Git submodules?

Reply:
Ah, indeed, we don't need it.  But I don't see any reason to drop it, as it still works fine even when the code is delivered by tarball, and if we ever switch back to git, it is important to have.  I can drop it if you think it's important to do so, though.

Reply:
It's not important as long as it works fine with tarball delivery.